### DISCUSSION TITLE
POST `/admin/identities` keep redirecting until "Maximmum number of redirects exceeded"

### DISCUSSION THREAD

**Discussion by NexZhu** [ory/kratos#3327] [Q&A] [answered]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/kratos/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/kratos/blob/master/CONTRIBUTING.md).
- [X] This issue affects my [Ory Network](https://www.ory.sh/) project.
- [X] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

POST `/admin/identities` keep redirecting until "Maximmum number of redirects exceeded"

### Reproducing the bug

Client is in Node.js, using `@ory/client` calling `createIdentity`,
Server responding 307 until client failed with "Maximmum number of redirects exceeded", see the log below.

### Relevant log output

```shell
Server log is the following repeated until the client stops with "Maximmum number of redirects exceeded":

kratos-kratos-1  | time=2023-06-19T09:14:28Z level=info msg=completed handling request http_request=map[headers:map[accept:application/json, text/plain, */* accept-encoding:gzip content-length:112 content-type:application/json user-agent:axios/0.21.4 x-forwarded-for:124.64.16.125 x-forwarded-host:kratos.demo.internetapi.cn x-forwarded-proto:https] host:kratos.example.com method:POST path:/admin/identities query:<nil> rem
ote:172.19.0.3:40524 scheme:http] http_response=map[headers:map[cache-control:private, no-cache, no-store, must-revalidate location:https://kratos.example.com/admin/identities vary:Origin] size:0 status:307 text_status:Temporary Redirect took:281.825?s]                            kratos-kratos-1  | time=2023-06-19T09:14:29Z level=info msg=started handling request http_request=map[headers:map[accept:application/json, text/plain, */* accept-encoding:gzip content-length:112 content-type:application/json user-agent:axios/0.21.4 x-forwarded-for:124.64.16.125 x-forward
ed-host:kratos.example.com x-forwarded-proto:https] host:kratos.example.com method:POST path:/admin/identities query:<nil> remote:172.19.0.3:40524 scheme:http]
```
```


### Relevant configuration

_No response_

### Version

Docker image: oryd/kratos:0.10.1

### On which operating system are you observing this issue?

Linux

### In which environment are you deploying?

Docker

### Additional Context

`@ory/client`: 0.2.0-alpha.48

**Comment by vinckr** [comment, marked as answer]
Hello @NexZhu 
Is this affecting your Ory Network project? If so could you please share the slug here or on the [Community Chat](https://slack.ory.sh/),
thanks!

**Reply by vinckr** [reply]
Would you be willing to share you solution here @NexZhu ? Or some details what the issue was =)
